README.md

```diff
@@ -18,24 +18,58 @@ Your group will be both a "Customer" and a "Designer".  First you'll prepare for
 
 ***Q1.1 Briefly describe your Mission:***
 
+The mission is to create a concept where the object solves a task where a screen is usually required.
```

Comment:
In the future, remember to fill out your names in the section above under "Part 0: Class Policies"!